static/app/views/issueDetails/quickTrace/issueQuickTrace.tsx

```diff
@@ -27,60 +18,23 @@ type Props = {
   quickTrace: undefined | QuickTraceQueryChildrenProps;
 };
 
-function TransactionMissingPlaceholder({
-  type,
-  event,
-}: {
-  event: Event;
-  type?: QuickTraceQueryChildrenProps['type'];
-}) {
-  useRouteAnalyticsParams({
-    trace_status: type === 'missing' ? 'transaction missing' : 'trace missing',
-  });
-
-  return (
-    <QuickTraceWrapper>
-      <QuickTraceContainer data-test-id="missing-trace-placeholder">
-        <EventNode
-          type="white"
-          icon={null}
-          tooltipProps={{isHoverable: true, position: 'bottom'}}
-          tooltipText={tct(
-            'The [type] for this event cannot be found. [link:Read the  docs] to understand why.',
-            {
-              type: type === 'missing' ? t('transaction') : t('trace'),
-              link: (
-                <ExternalLink href="https://docs.sentry.io/product/sentry-basics/tracing/trace-view/#troubleshooting" />
-              ),
-            }
-          )}
-        >
-          ???
-        </EventNode>
-        <TraceConnector />
-        <EventNode type="error" data-test-id="event-node">
-          <ErrorNodeContent>
-            <IconFire size="xs" />
-            {t('This Event')}
-          </ErrorNodeContent>
-        </EventNode>
-        <TraceLink event={event} />
```

Comment:
One other note is that this `<TraceLink />` component can render a useful "View full trace" link when a trace exists but a transaction does not. I don't know if we care about showing that or not, but just wanted to point out that this does sometimes render something useful.

Reply:
good catch - that seems useful to keep in since it has its own check to see if it should show a link